packages/nimble-components/src/icon-base/tests/icon-metadata.ts

```diff
@@ -457,6 +466,9 @@ export const iconMetadata: {
     IconOutwardSquaresThree: {
         tags: ['undeploy']
     },
+    IconPageListCog: {
```

Comment:
Looks like Font Awesome [would have this](https://fontawesome.com/search?q=list&o=r&s=solid&ip=classic) as `IconSquareListGear`, but Nimble has consistently used `cog` instead of `gear`. 

So, I'd propose `IconSquareListCog`.